avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1569,6 +1579,9 @@ def exportFields(cfgGen, Tsave, Fields, relFile, demOri, outDir, logName):
     resTypesString = cfgGen['resType']
     resTypes = resTypesString.split('_')
     tSteps = fU.getTimeIndex(cfgGen, Fields)
+    if -1 not in tSteps:
+        tSteps.append(-1)
+        log.info('-1 added to tStep')
```

Comment:
What is this for?

Reply:
that in any case, so whatever inputs you provide in the ini file for tSteps (the time steps that you want data to be exported to Outputs), the last time step is exported